index.js

```diff
@@ -405,7 +411,7 @@ BFFS.prototype.publish = function publish(spec, options, fn) {
             extension: file.extension,
             filename: file.filename,
             fingerprint: print,
-            url: file.url
+            url: gz ? file.url : file.compressedUrl
```

Comment:
This seems reversed to me. If `gz` is `false` we want the `compressedUrl`?

Reply:
@fritzmonkey thanks, you are correct